src/main/java/gov/usds/case_issues/controllers/CaseDetailsApiController.java

```diff
@@ -69,11 +77,25 @@ public CaseDetails getCaseDetails(@PathVariable String caseManagementSystemTag,
 		return ResponseEntity.ok(replacement);
 	}
 
-	@PostMapping("activeSnooze/notes")
+	@GetMapping("attachments")
+	public ResponseEntity<List<AttachmentSummary>> getAttachments(@PathVariable String caseManagementSystemTag,
+			@PathVariable String receiptNumber) {
+		TroubleCase rootCase = _caseDetailsService.findCaseByTags(caseManagementSystemTag, receiptNumber);
+		List<AttachmentSummary> attachments = toSummaries(_attachmentService.findAttachmentsForCase(rootCase));
+		return ResponseEntity.ok(attachments);
+	}
+
+	@PostMapping({"activeSnooze/notes","activeSnooze/attachments"})
 	@RequireUpdateCasePermission
-	public ResponseEntity<?> addNote(@PathVariable String caseManagementSystemTag,
-			@PathVariable String receiptNumber, @RequestBody AttachmentRequest newNote) {
-		_caseDetailsService.annotateActiveSnooze(caseManagementSystemTag, receiptNumber, newNote);
-		return ResponseEntity.accepted().build();
+	public ResponseEntity<List<AttachmentSummary>> addAttachment(@PathVariable String caseManagementSystemTag,
+			@PathVariable String receiptNumber, @RequestBody AttachmentRequest attachment) {
+		List<CaseAttachmentAssociation> allAttachments = _caseDetailsService.annotateActiveSnooze(caseManagementSystemTag, receiptNumber, attachment);
+		return ResponseEntity.accepted().body(toSummaries(allAttachments));
```

Comment:
I think this should only return the singular object created. With the extra data that the frontend does not have. Such as `id`, `created_at`, etc.

Returning the list of attachments would make it hard for the frontend to reliably determine the correct id of the note that was created

Reply:
That seems objectively correct. Right now there's nothing for the UI to do with an ID, and I don't think we provide one, but we should provide one so that eventually there can be (mostly DELETE).

Options:
* ID is specific to the case, so manipulation URLs (when they exist) would be along the lines of `/${CASE}/currentSnooze/attachments/${ID}`, and a given ID would correspond to the underlying attachment. This would allow some smarts on the client side if we wanted to associate a particular attachment to a new case, and already knew all of its details. Mechanically, this would be using the internal ID of the CaseAttachment itself.
* ID is globally unique (mechanically, the internal ID of the AttachmentAssociation object), so a URL could be just `/attachments/${ID}` if we wanted it to, or it could be `/$CASE/attachments/${ID}` and we'd have some redundancy (more opportunities to catch client errors?)

I'm inclined toward the first one—I like the idea of being able to undelete a previously-deleted note with fairly little fuss, for one thing—but I could be persuaded that this is wrong.

Reply:
First one seems good to me

Reply:
Done. I think. I should probably add a test for that specific field...